### PR TITLE
add readme for documentdb starter, and spring-data-documentdb

### PR DESCRIPTION
**Pull request by yungez**
@ZhijunZhao @aovechki

### CODE REVIEW

File: spring-data-azure-documentdb/README.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+## Overview
+Azure DocumentDB Spring Data provides initial Spring Data support for [Azure Cosmos DB Document API](https://docs.microsoft.com/en-us/azure/cosmos-db/documentdb-introduction) based on Spring Data framework. Key functionality supports so far including save, delete and find. More features will coming soon.
+
```
Comment (ZhijunZhao) [inline spring-data-azure-documentdb/README.md]: functionalities?

supports -> supported

including -> include?

File: spring-data-azure-documentdb/README.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+## Sample Codes
+Pls refer to [sample projects here](../spring-data-azure-documentdb-sample).
```
Comment (ZhijunZhao) [inline spring-data-azure-documentdb/README.md]: codes -> code

File: spring-data-azure-documentdb/README.md (hunk 1 above)
Comment (yungez) [inline spring-data-azure-documentdb/README.md]: resolved

File: spring-data-azure-documentdb/README.md (hunk 2 above)
Comment (yungez) [inline spring-data-azure-documentdb/README.md]: resolved

**Pull request merged**